src/scripts/clipperUI/componentBase.ts

```diff
@@ -105,6 +105,27 @@ export abstract class ComponentBase<TState, TProps> {
 				} else if (e.which === Constants.KeyCodes.tab) {
 					// Since they are using the keyboard, revert to the default value of the outline so it is visible
 					element.style.outlineStyle = "";
+				} else if (e.which === Constants.KeyCodes.up) {
+					if (element.hasAttribute("data-" + Constants.AriaSet.setNameForArrowKeyNav)) {
+						let posInSet = parseInt(element.getAttribute("aria-posinset"), 10);
+						if (posInSet === 1) {
+							return;
+						}
+						const buttons = document.querySelectorAll("a[data-" + Constants.AriaSet.setNameForArrowKeyNav + "=" + valueOfSetNameForArrowKeyNav + "]");
```

Comment:
This code only works if the 'buttons' are html 'a' tags. We should allow this code to work for all elements.